<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginAuth</class>
 <widget class="QDialog" name="LoginAuth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Authentication</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: grey</string>
  </property>
  <widget class="QLineEdit" name="UsernameField">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>50</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #999999</string>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordField">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>90</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #999999</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="WindowLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login Authentication</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="FailedAuth">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Authentication failed.  Try again.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
